assignment2/employee.rb

```diff
@@ -0,0 +1,21 @@
+class Employee
+    def initialize(emp_experience)
+        @emp_experience = emp_experience
```

Comment:
initialize method should get all the employee details lilke name, age, etc.
Try to use attr_accessor here for employee attributes.